builtin/providers/aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -294,6 +311,9 @@ func flattenCacheBehavior(cb *cloudfront.CacheBehavior) map[string]interface{} {
 	if len(cb.TrustedSigners.Items) > 0 {
 		m["trusted_signers"] = flattenTrustedSigners(cb.TrustedSigners)
 	}
+	if len(cb.LambdaFunctionAssociations.Items) > 0 {
+		m["lambda_function_association"] = flattenLambdaFunctionAssociations(cb.LambdaFunctionAssociations)
```

Comment:
no, it's a key in the schema. If it were plural then the config to use it would be 

```
lambda_funcation_associations {
  event_type = "blah"
  arn = "blah"
}
lambda_funcation_associations {
  event_type = "blah"
  arn = "blah"
}
...
```